server/server-wildfly/src/main/java/org/jboss/aerogear/sync/server/wildfly/SyncEndpoint.java

```diff
@@ -70,13 +74,13 @@ public String onMessage(String message, Session webSocketSession) {
     }
 
     @OnOpen
-    public void myOnOpen(Session session) {
-        System.out.println("WebSocket opened: " + session.getId());
+    public void myOpen(Session session) {
+        logger.info("WebSocket opened: " + session.getId());
     }
 
     @OnClose
-    public void myOnClose(CloseReason reason, Session webSocketSession) {
-        System.out.println("Closing a WebSocket due to " + reason.getReasonPhrase());
+    public void myClose(CloseReason reason, Session webSocketSession) {
```

Comment:
lol

yeah :-)

On Friday, February 27, 2015, Hoyt Summers Pittman <notifications@github.com>
wrote:

> In
> server/server-wildfly/src/main/java/org/jboss/aerogear/sync/server/wildfly/SyncEndpoint.java
> <https://github.com/secondsun/aerogear-sync-server/pull/1#discussion_r25510603>
> :
>
> >      }
> >
> >      @OnClose
> > -    public void myOnClose(CloseReason reason, Session webSocketSession) {
> > -        System.out.println("Closing a WebSocket due to " + reason.getReasonPhrase());
> > +    public void myClose(CloseReason reason, Session webSocketSession) {
>
> maybe onClose instead of myClose?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/secondsun/aerogear-sync-server/pull/1/files#r25510603>
> .
>


-- 
Sent from Gmail Mobile